lambda-bedrock-dynamodb-sam/template.yaml

```diff
@@ -0,0 +1,78 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Sam for Bedrock and Dynamodb pattern
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 180
+    MemorySize: 128
+
+    # You can add LoggingConfig parameters such as the Logformat, Log Group, and SystemLogLevel or ApplicationLogLevel. Learn more here https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-function.html#sam-function-loggingconfig.
+    # LoggingConfig:
+    #   LogFormat: JSON
+Parameters:
+  tableName:
+    Type: String
+    Description: Dynamodb table name
+Resources:
+  # Creates dynamodb table 
+  Outputtable:
+    Type: AWS::DynamoDB::Table
+    Properties: 
+      TableName: !Sub "${tableName}"
+      AttributeDefinitions: 
+        - AttributeName: id
+          AttributeType: S
+      KeySchema: 
+        - AttributeName: id
+          KeyType: HASH
+      ProvisionedThroughput: 
+        ReadCapacityUnits: 1
+        WriteCapacityUnits: 1
+ 
+  # Creates Lambda function 
+
+  InvokeBedrock:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      FunctionName: lambda-processor
+      CodeUri: src/
+      Handler: app.lambda_handler
+      Environment:
+        Variables:
+          table_name: !Ref Outputtable
```

Comment:
add a variable for model name so users can easily test different models.

Reply:
model variable added